CODEOWNERS

```diff
@@ -1,12 +1,9 @@
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 
 # SECTION: Owner(s) for everything in the repo, unless a later match takes precedence
-#          **************************************************************************
-#          ***                    NO GLOBAL OWNER(S) SPECIFIED                    ***
-#          ***  Ideally this will be defined for a healthy, well-maintained repo  ***
-#          **************************************************************************
 # FILE PATTERN                              OWNER(S)
-*                                           @NONE
+*                                           @alxwr
+*                                           @noelmcloughlin
```

Comment:
Multiple entries are handled like this:

* https://github.com/saltstack-formulas/postgres-formula/blob/1c0657b17481ad5633c69e322a48af67dc16f2f7/CODEOWNERS#L5

Reply:
okay, will fix